CMakeLists.txt

```diff
@@ -255,9 +255,11 @@ IF (WIN32 AND WINHTTP)
 
 		INCLUDE_DIRECTORIES(deps/winhttp)
 		LINK_DIRECTORIES(${LIBWINHTTP_PATH})
+		LIST(APPEND LIBGIT2_PC_LIBS "-L${LIBWINHTTP_PATH}")
```

Comment:
I'm not 100% sure about this line here. I think it's wrong to specify this here, but maybe someone else knows better. :smile: